fix #2627 by hardcoding links

motatoes opened the pull request:
The following links have been updated in the README file to point to absolute links

- docs/source/notebooks/getting_started.ipynb -> http://docs.pymc.io/notebooks/getting_started
- notebooks/api_quickstart.ipynb -> http://docs.pymc.io/notebooks/api_quickstart
- notebooks/getting_started.ipynb -> http://docs.pymc.io/notebooks/getting_started
- /examples -> http://docs.pymc.io/examples
- /api -> http://docs.pymc.io/api

motatoes committed:
fix #2627 by hardcoding links

motatoes committed:
fix #2627 by hardcoding links

springcoil commented:
Hey @motatoes could you add a description. For this?

springcoil reviewed (changes requested):
LGTM I'd just add a little description and a link to the issue that referenced this for clarity.

motatoes committed:
remove :refs: from links

motatoes committed:
remove :refs: from links

motatoes committed:
remove :refs: from links

twiecki commented on README.rst:
```diff
@@ -10,8 +10,11 @@ which focuses on advanced Markov chain Monte Carlo and variational fitting
 
-Check out the `getting started guide <docs/source/notebooks/getting_started.ipynb>`__!
-
+<<<<<<< HEAD
```
These shouldn't show up and are the result of a merge-conflict.

motatoes replied:
@twiecki  yes sorry did a mistake by forgetting to save the file locally. My next commit in this branch removes these lines

motatoes committed:
Merge branch 'fix-docs-404' of github.com:motatoes/pymc3 into fix-docs-404

motatoes commented:
Hi @springcoil I've added a description of the updated links to the readme file as requested. 

I added a few extra commits as well because the .rst links were not rendering correctly. I couldn't squash the commits together because I was having issues with rebase.

springcoil commented:
Ok someone will help you out.

On 24 Oct 2017 9:39 pm, "Mohamed" <notifications@github.com> wrote:

> Hi @springcoil <https://github.com/springcoil> I've added a description
> of the updated links to the readme file as requested.
>
> I added a few extra commits as well because the .rst links were not
> rendering correctly. I couldn't squash the commits together because I was
> having issues with rebase.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pymc-devs/pymc3/pull/2657#issuecomment-339124014>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AA8DiALxDQVwIe02tFRClMjmrUNjynGZks5svksagaJpZM4QE-s9>
> .
>

twiecki commented:
Thanks @motatoes!
